platform/README.mdx

```diff
@@ -68,7 +68,7 @@ If for any reason, you cannot have the Smallstep Agent on your devices, Smallste
 
 Smallstep integrates with your MDM to deploy client certificates to company-managed devices to enable certificate-based network authentication for Wi-Fi (802.1x EAP-TLS WPA-Enterprise), VPN, ZTNA, etc.
 
-We offer integrations for any MDMs for Apple and Windows devices that support Dynamic SCEP like Jamf, Intune, Workspace ONE, Mosyle, Ivanti, e.t.c.
+We offer integrations for any MDMs for Apple, Windows, and ChromeOS devices that support Dynamic SCEP like Jamf, Intune, Workspace ONE, Mosyle, Ivanti, and Google Workspace.

```

Comment:
This is in the context of dynamic SCEP. Technically, we can do it and have tested it, but I don't think the current implementation is something we want to support officially. We would likely want to iterate some more on it properly, if users need it.